org.eclipse.jdt.core/model/org/eclipse/jdt/core/compiler/CompilationParticipant.java

```diff
@@ -143,6 +151,32 @@ public void processAnnotations(BuildContext[] files) {
 	// do nothing by default
 }
 
+/**
+ * Notifies this participant about a finished class compilation. This notification occurs right before the compiled
+ * class is written out to disk. This allows to perform arbitrary byte code manipulation on the generated class.
+ *
+ * If additional dependencies to other classes are introduced by the modified class, these dependencies have to be
+ * recorded by calling org.eclipse.jdt.core.compiler.BuildContext.recordDependencies(String[]).
+ *
+ * In case problems occur, they can be reported using
+ * org.eclipse.jdt.core.compiler.BuildContext.recordNewProblems(CategorizedProblem[]).
+ *
+ * The received class data could have already been transformed by another compilation participant. If the compilation
+ * participants have to be called in a specific order, the attribute "requires" can be set on the extension point
```

Comment:
"can be set on the extension point entries" -> "can be used in the participant contribution entry"